Arduino/MiP_ProMini_Pack_Library/README.md

```diff
@@ -116,7 +117,7 @@ Clap            | [enableClapEvents()](#enableclapevents)
 Shake           | [hasBeenShaken()](#hasbeenshaken)
 Version Info    | [readSoftwareVersion()](#readsoftwareversion)
 <br>            | [readHardwareInfo()](#readhardwareinfo)
-
+App             | [disconnectApp()](#disconnectapp)
```

Comment:
Would probably be ok to add to the initialization section as it is similar to end().

Reply:
Agreed, will move.